docker/database.mysql.yml

```diff
@@ -0,0 +1,21 @@
+# SQLite version 3.x
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem 'sqlite3'
```

Comment:
Interesting comment, especially when reading it in database.mysql.yml and database.postgresql.yml ;-)